src/libvirtApi/helpers.js

```diff
@@ -41,11 +41,13 @@ export const Enum = {
     VIR_DOMAIN_UNDEFINE_MANAGED_SAVE: 1,
     VIR_DOMAIN_UNDEFINE_SNAPSHOTS_METADATA: 2,
     VIR_DOMAIN_UNDEFINE_NVRAM: 4,
-    VIR_DOMAIN_SNAPSHOT_LIST_INTERNAL: 256,
+    // https://libvirt.org/html/libvirt-libvirt-domain-snapshot.html
     VIR_DOMAIN_SNAPSHOT_REVERT_RUNNING: 1,
     VIR_DOMAIN_SNAPSHOT_REVERT_PAUSED: 2,
     VIR_DOMAIN_SNAPSHOT_REVERT_FORCE: 4,
     VIR_DOMAIN_SNAPSHOT_REVERT_RESET_NVRAM: 8,
+    VIR_DOMAIN_SNAPSHOT_CREATE_DISK_ONLY: 16,
+    VIR_DOMAIN_SNAPSHOT_LIST_INTERNAL: 256,
```

Comment:
Hmm, does that mean we can only list internal snapshots?

Reply:
We don't actually use that flag. I just moved it a couple of lines down to keep the flags sorted by number.

Reply:
I dropped it now, no point in having it if we don't use it.